scripts/sanitycheck

```diff
@@ -2314,8 +2314,8 @@ class TestSuite:
                     rowdict["passed"] = True
                     if goal.handler:
                         rowdict["handler_time"] = goal.metrics["handler_time"]
-                    rowdict["ram_size"] = goal.metrics["ram_size"]
-                    rowdict["rom_size"] = goal.metrics["rom_size"]
+                        rowdict["ram_size"] = goal.metrics["ram_size"]
+                        rowdict["rom_size"] = goal.metrics["rom_size"]
```

Comment:
ram/rom sizes do not depend on a handler, they can be extracted with just building, so this can't be correct. What issue are you trying to fix and how can we reproduce it?